blockchain/msgs_test.go

```diff
@@ -86,3 +90,50 @@ func TestBcStatusResponseMessageValidateBasic(t *testing.T) {
 		})
 	}
 }
+
+func TestBlockchainMessageVectors(t *testing.T) {
+
+	block := types.MakeBlock(int64(3), []types.Tx{types.Tx("Hello World")}, nil, nil)
+
+	bpb, err := block.ToProto()
+	require.NoError(t, err)
+
+	testCases := []struct {
+		testName string
+		bmsg     proto.Message
+		expBytes []byte
+	}{
+		{"BlockRequestMessage", &bcproto.BlockRequest{Height: 1}, []byte{0x8, 0x1}},
+		{"BlockRequestMessage", &bcproto.BlockRequest{Height: math.MaxInt64},
+			[]byte{0x8, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x7f}},
+		{"BlockResponseMessage", &bcproto.BlockResponse{Block: bpb}, []byte{0xa, 0x6c, 0xa, 0x37, 0xa, 0x0, 0x18,
```

Comment:
how one can generate these bytes?

Reply:
the ones in `[]byte{..}`? this is the output of using `require.Equal` and it failing. `fmt.Println(..)` outputs it into numbers. This is to keep consistent with how it is done in vote test vectors.